lib/morse_translator.rb

```diff
@@ -40,4 +42,32 @@ def initialize
                     " " => " "}
   end
 
+  def translate(string)
+    morse_code = ""
+    string.downcase.split("").each do |letter|
+      morse_code << @dictionary[letter]
+    end
+    return morse_code
+  end
+
+  def morse_to_eng(morse_code)
+    string = ""
+    morse_code.split("  ").each do |code|
+      if code == morse_code.split("  ").last
+        string << code_to_word(code)
+      else
+        string << code_to_word(code) + " "
+      end
+    end
+    return string
```

Comment:
Here's a suggestion... instead of keeping all of these words in a string, put them in an array and then use `.join(" ")` to put the spaces in. This would eliminate the need for an if inside your each.